src/uwtools/config/jinja2.py

```diff
@@ -166,15 +166,19 @@ def render(
     dry_run: bool = False,
 ) -> bool:
     """
-    Render a Jinja2 template.
+    Check and render a Jinja2 template.
 
     :param input_file: Path to the Jinja2 template file to render.
     :param output_file: Path to the file to write the rendered Jinja2 template to.
     :param values_file: Path to the file supplying values to render the template.
     :param keq_eq_val_pairs: "key=value" strings to supplement values-file values.
     :param values_needed: Just issue a report about variables needed to render the template?
     :param dry_run: Run in dry-run mode?
+    :return: Jinja2 template was successfully rendered.
     """
+
+    # Render template.
+
     _report(locals())
     values = _set_up_values_obj(
```

Comment:
These steps are needed in all instances. I did not create a separate helper function for this section because of the number of arguments and returns that would need to be passed, but I'm not sure that was the right move.